mlir/test/mlir-miopen-driver/e2e/conv2d_host_validation_yxck_nhwc_nhwk2.mlir

```diff
@@ -0,0 +1,20 @@
+// COM: yxck/nhwc/nhwk
```

Comment:
Could you summarize the difference and similarities between:
- conv2d_host_validation_yxck_nhwc_nhwk1.mlir 
- conv2d_host_validation_yxck_nhwc_nhwk2.mlir 

Such that it becomes more apparent what you are testing by looking at the filename